app/src/main/res/layout/challenge_viewholder_layout.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
this layout is for the challenge item, not the view holder. Can we rename this to better reflect what it represents?